dagger/service/services.py

```diff
@@ -691,7 +691,7 @@ async def _invoke_store_get_value_for_key_with_timer(self, key: str):
         start_time = self.faust_app.loop.time()
         random: Random = Random()
         random.seed(str(key))
-        partition: int = random.randint(0, Dagger.LOCK_STRIPE_SIZE)  # nosec
+        partition: int = random.randint(0, Dagger.LOCK_STRIPE_SIZE - 1)  # nosec
```

Comment:
To avoid this nasty `- 1`, you can use `random.randrange(Dagger.LOCK_STRIPE_SIZE)` instead

Reply:
oh yeah! thanks @JeffreyGreenII